src/main/java/Parser.java

```diff
@@ -0,0 +1,80 @@
+import task.TaskList;
```

Comment:
Parser is a good name for a class responsible for parsing input. It clearly describes its purpose. Well done here!